solc_select/__main__.py

```diff
@@ -61,14 +61,19 @@ def solc_select() -> None:
         switch_global_version(args.get(USE_VERSION), args.get("always_install"))
 
     elif args.get(SHOW_VERSIONS) is not None:
-        res = current_version()
-        if res:
-            (current_ver, source) = res
-        for version in reversed(sorted(installed_versions())):
-            if res and version == current_ver:
-                print(f"{version} (current, set by {source})")
-            else:
-                print(version)
+        if installed_versions():
```

Comment:
Can you store the result and not call `installed_versions` twice

Reply:
fixed! thanks